packages/harpguru-core/src/hooks/use-octave-column-groups/use-octave-column-groups.test.ts

```diff
@@ -1,14 +1,18 @@
 import { useGlobal } from 'reactn'
 
-import { buildMockUseGlobalImplementation } from '../../../../test-resources'
+import { buildMockUseGlobalImplementation } from '../../test-resources'
 
 import { useOctaveColumnGroups } from './use-octave-column-groups'
```

Comment:
We have moved the `useOctaveColumnGroups` hook to the project root hooks folder, but it turns out that that wasn't actually necessary. Move it back to the `harp-face` folder.

Reply:
Done